client/tests/test_executor_resources.py

```diff
@@ -10,12 +10,12 @@
         ("a", "b", "", "","c"),
         ("", "", "a", "b","c"),
         pytest.param("a", "b", "a","b","c", marks=pytest.mark.xfail),
-        pytest.param("", "", "", "","c", marks=pytest.mark.xfail)
+        pytest.param("", "", "", "","c", marks=pytest.mark.xfail),
         pytest.param("a", "b", "","","", marks=pytest.mark.xfail)
 
     ]
 )
-def test_databricks_credentials(username, password, host, token):
+def test_databricks_credentials(username, password, host, token, cluster_id):
```

Comment:
@sdreyer, these are the syntax errors that were preventing me from discovering pytests in VS Code. Let me know if you'd like me to open up a separate PR to `main` for this fix if you don't feel it should be included with this PR.

Reply:
This is fine